phys/module_sf_bem.F

```diff
@@ -1330,8 +1585,8 @@ subroutine algebra_long(emwal,emwin,sigma,twalint,twinint,&
 	real b_wind(6)
 	real emwal_av		!averadge emissivity of the wall
 	real emwin_av		!averadge emissivity of the window
-	real em_av		!averadge emissivity
-        real twal_int(6)        !twalint 
+   	real em_av		!averadge emissivity
+            real twal_int(6)        !twalint 
```

Comment:
indent increased

Reply:
Jimy,
I lined up these also. There were tabs vs spaces used here. I have followed the existing convention, and they should line up nicely now.

Reply:
Jimy,
Here is what the source code looks like in an editor on my Mac:
```
        integer iw,jw
        real b_wall(6)  
        real b_wind(6)
        real emwal_av           !averadge emissivity of the wall
        real emwin_av           !averadge emissivity of the window
        real em_av              !averadge emissivity
        real twal_int(6)        !twalint 
        real twin(4)            !twinint 
```

The github representation is playing with tabs and spaces. My editor will not allow me to introduce a tab character in the source code.